README.md

```diff
@@ -806,6 +795,145 @@ VIDEO_FACIAL_CAPTURE_STEP_2 - User reached the 2nd challenge during "liveness vi
 UPLOAD - User's file is uploading
 ```
 
+## Premium Enterprise Features
+
+These features must be enabled for your account before they can be used. For more information, please contact your Onfido Solution Engineer or Customer Success Manager.
+
+### Customized API Requests
+
+This premium enterprise feature enables you to control the data collected by the Onfido SDK through the use of callbacks that are invoked when the user submits their captured media. These callbacks provide all of the information that would normally be sent directly to the Onfido API and expect a promise in response that controls what the SDK does next. Before the feature can be used, it must be enabled for your account. Once enabled, you will need to set `useCustomizedApiRequests` to `true` and provide the callbacks for `onSubmitDocument` and `onSubmitSelfie` within the `enterpriseFeatures` block of the configuration options. The callback for video is not officially supported yet.
+
+Example:
+
+```javascript
+Onfido.init({
+  // Other options here
+  enterpriseFeatures: {
+    useCustomizedApiRequests: true,
+    onSubmitDocuments: (documentData) => {
+      // Your callback code here
+    },
+    onSubmitSelfie: (selfieData) => {
+      // Your callback code here
+    },
+  },
+})
+```
+
+To enable callbacks on the cross-device flow you must also host the cross-device experience of the Onfido SDK yourself. This can be done using the [cross device URL](#cross-device-url) premium enterprise feature. Once this is set up you must initialize the SDK with the callbacks and `useCustomizedApiRequests` options shown above as well as `mobileFlow: true`.
+
+#### Callbacks Overview
+
+The callbacks will provide you with a FormData object including the information that the SDK would send to Onfido. These callbacks will be invoked when the user confirms their image on the UI and won’t send the request to Onfido unless requested in the response.
+
+**onSubmitDocument FormData Paramaters**
+
+```javascript
+{
+  file: blob,
+	side: string,
+	type: string,
+	sdk_validations: {
+ 		detect_document: string,
+		detect_cutoff: string,
+    detect_glare: string,
+    detect_blur: string,
+  },
+  sdk_source: string,
+  sdk_version: string,
+  sdk_metadata: {
+    captureMethod: string,
+    imageResizeInfo: string | null,
```

Comment:
`imageResizeInfo` is actually an object if provided, e.g. 
```
"imageResizeInfo": {
    "resizedFrom":
    {
        "width": 7016,
        "height": 4951,
        "fileSize": 10544735
    },
    "resizedTo": {
        "width": 1440,
        "height": 1016.1687571265678,
        "fileSize": 426058
    }
}
```

Reply:
Thanks for pointing this out. I made the change but it's been decided now that we should just have sdk_metadata as `object` so I've changed it to that now.